Python/Batch/sample4_job_scheduler.py

```diff
@@ -0,0 +1,177 @@
+# Sample4_job_scheduler.py Code Sample
+
+# Create a job schedule
+#    With job specification that has an autopool
+#    autopool has a start task
+#    job specification has a job manager task
+
+
+from __future__ import print_function
+
+try:
+    import configparser
+except ImportError:
+    import ConfigParser as configparser
+
+import datetime
+import os
+
+import azure.batch.batch_service_client as batch
+import azure.storage.blob as azureblob
+import azure.batch.batch_auth as batchauth
+import azure.batch.models as batchmodels
+
+import common.helpers
+
+_CONTAINER_NAME = 'jobscheduler'
+_SIMPLE_TASK_NAME = 'simple_task.py'
+_SIMPLE_TASK_PATH = os.path.join('resources', 'simple_task.py')
+_PYTHON_DOWNLOAD = 'https://www.python.org/ftp/python/3.7.3/python-3.7.3-amd64.exe'
+_PYTHON_INSTALL = '.\python373.exe /passive InstallAllUsers=1 PrependPath=1 Include_test=0'
+_USER_ELEVATION_LEVEL = 'admin'
+
+
+def create_job_schedule(batch_client, job_id, vm_size, vm_count, block_blob_client):
+    cloud_service_config = batchmodels.CloudServiceConfiguration(os_family='6')
+
+    user_id = batchmodels.UserIdentity(
+        auto_user=batchmodels.AutoUserSpecification(elevation_level=_USER_ELEVATION_LEVEL))
+
+    python_download = batchmodels.ResourceFile(http_url=_PYTHON_DOWNLOAD, file_path='python373.exe')
+
+    pool_info = batchmodels.PoolInformation(
+        auto_pool_specification=batchmodels.AutoPoolSpecification(
+            auto_pool_id_prefix="JobScheduler",
+            pool=batchmodels.PoolSpecification(
+                vm_size=vm_size,
+                target_dedicated_nodes=vm_count,
+                cloud_service_configuration=cloud_service_config,
+                start_task=batchmodels.StartTask(
+                    command_line=common.helpers.wrap_commands_in_shell('windows', ['{}'.format(_PYTHON_INSTALL)]),
+                    resource_files=[python_download],
+                    wait_for_success=True,
+                    user_identity=user_id)),
+            keep_alive=False,
+            pool_lifetime_option=batchmodels.PoolLifetimeOption.job_schedule))
+
+    sas_url = common.helpers.upload_blob_and_create_sas(
+        block_blob_client,
+        _CONTAINER_NAME,
+        _SIMPLE_TASK_NAME,
+        _SIMPLE_TASK_PATH,
+        datetime.datetime.utcnow() + datetime.timedelta(hours=1))
+
+    job_spec = batchmodels.JobSpecification(
+        pool_info=pool_info,
+        on_all_tasks_complete=batchmodels.OnAllTasksComplete.terminate_job,
```

Comment:
Maybe a comment here as well, explaining how this is important so that the schedule will recur (since if jobs don't end new ones can't be created under a schedule?)